addon/mixins/oauth2-implicit-grant-callback-route-mixin.js

```diff
@@ -0,0 +1,57 @@
+import Ember from 'ember';
+
+const { inject: { service }, Mixin } = Ember;
+
+export default Mixin.create({
```

Comment:
This needs documentation - have a look at e.g. the `AuthenticatedRouteMixin` for reference.